app/src/main/res/values-sw600dp-land/dimens.xml

```diff
@@ -295,6 +295,17 @@
   <dimen name="topic_practice_footer_view_margin_top">24dp</dimen>
   <dimen name="topic_practice_footer_view_margin_bottom">0dp</dimen>
 
+  <!-- Completed Story Item -->
+  <dimen name="completed_story_item_topic_container_marginStart">32dp</dimen>
+  <dimen name="completed_story_item_topic_container_marginEnd">24dp</dimen>
+  <dimen name="completed_story_item_topic_container_marginTop">32dp</dimen>
```

Comment:
Please add bottom margin as 0dp otherwise it will end up using value from layout/dimens.xml file which is 12dp.